resharper/src/resharper-unity/ShaderLab/Psi/Resolve/IVariableReferenceReference.cs

```diff
@@ -0,0 +1,9 @@
+﻿using JetBrains.ReSharper.Psi.Resolve;
+
+namespace JetBrains.ReSharper.Plugins.Unity.ShaderLab.Psi.Resolve
+{
+    // TODO: Terrible name...
```

Comment:
It can use a sample of the entity it is describing in the comment then :)
Also, WDYT something like IReferenceToVariableReference (doesn't sound good to me too though)?